### PR TITLE
Add wilight switch

### PR DESCRIPTION
**Pull request by leofig-rj**
Included switch / irrigation integration.

## Proposed change

Add Documentation for WiLight Switch

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/62873
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/wilight.markdown [hunk 1]
```diff
@@ -25,23 +28,59 @@ The `wilight` integration is to integrate [WiLight](http://www.wilight.com.br) d
...
+- [Fan](#fan) (WiLight model V-104).
+- Light (WiLight model I-100, I-102, I-107, I-110 and I-112).
+- [Irrigation](#irrigation) / Switch (WiLight model R-105).
 
 {% include integrations/config_flow.md %}
 
 ## Fan
 
-The `wilight` integration allows you to control your Fans from within Home Assistant.
+The `wilight` `fan` integration allows you to control your Fans from within Home Assistant.
```
Comment (MartinHjelmare) [inline source/_integrations/wilight.markdown]: The integration is wilight. The integration has a fan platform. But I don't think we need to specify that to users. It's enough to say that it allows users to control their wilight fan.

Comment (MartinHjelmare) [reply source/_integrations/wilight.markdown]:
```suggestion
The `wilight` integration allows you to control your Fans from within Home Assistant.
```

File: source/_integrations/wilight.markdown [hunk 2]
```diff
@@ -25,23 +28,59 @@ The `wilight` integration is to integrate [WiLight](http://www.wilight.com.br) d
...
 
-### Services
+### Fan Services
 
-There are several services which can be used for automations and control of the fan:
+There are several services which can be used for automations and control of the `fan`:
```
Comment (MartinHjelmare) [inline source/_integrations/wilight.markdown]:
```suggestion
There are several services which can be used for automations and control of the fan:
```

File: source/_integrations/wilight.markdown [hunk 3]
```diff
@@ -25,23 +28,59 @@ The `wilight` integration is to integrate [WiLight](http://www.wilight.com.br) d
...
-| `turn_on` | Calling this service will turn the fan on and set the speed and direction to the last used ones (defaults to high and forward, entity_id is required).
+| `set_percentage` | Calling this service sets the fan speed (`entity_id` and `percentage` are required parameters). Percentage must be in the range 0-100, percentage = 0 turns off the fan. There are three speed levels 33%, 66% and 100%. Adjusting to intermediate levels goes to the next higher level.
+| `set_direction` | Calling this service will set the fan direction (`entity_id` and `direction` are required parameters, and direction must be one of the following: forward or reverse). Calling this service will turn the fan on.
+| `toggle` | Calling this service will toggle the fan between on and off states (`entity_id` is required).
+| `turn_off` | Calling this service will turn the fan off (`entity_id` is required).
+| `turn_on` | Calling this service will turn the fan on and set the speed and direction to the last used ones (defaults to high and forward, `entity_id` is required).
+
+## Irrigation
+
+The `wilight` `switch` integration allows you to control your Irrigation from within Home Assistant.
```
Comment (MartinHjelmare) [inline source/_integrations/wilight.markdown]:
```suggestion
The `wilight` integration allows you to control your Irrigation from within Home Assistant. This is exposed via switches in Home Assistant.
```

File: source/_integrations/wilight.markdown [hunk 4]
```diff
@@ -25,23 +28,59 @@ The `wilight` integration is to integrate [WiLight](http://www.wilight.com.br) d
...
+There are two switch types in Irrigation: `watering switch` and `pause switch`.
```
Comment (MartinHjelmare) [inline source/_integrations/wilight.markdown]:
```suggestion
There are two switch types for Irrigation: `watering switch` and `pause switch`.
```

File: source/_integrations/wilight.markdown [hunk 5]
```diff
@@ -25,23 +28,59 @@ The `wilight` integration is to integrate [WiLight](http://www.wilight.com.br) d
...
+`Watering switch` turn on and off the irrigation valve, while the `pause switch` disable / enable the action of `watering switch`.
```
Comment (MartinHjelmare) [inline source/_integrations/wilight.markdown]:
```suggestion
A `watering switch` can turn on and off the irrigation valve, while the `pause switch` can disable / enable the action of a `watering switch`.
```

File: source/_integrations/wilight.markdown [hunk 6]
```diff
@@ -25,23 +28,59 @@ The `wilight` integration is to integrate [WiLight](http://www.wilight.com.br) d
...
+`Triggers` activate (turn on) the `watering switch` (irrigation valve) at the programmed time, which can be defined on which days of the week it is desired or if only once (today).
```
Comment (MartinHjelmare) [inline source/_integrations/wilight.markdown]:
```suggestion
`Triggers` activate (turn on) the `watering switch` (irrigation valve) at the programmed time. The trigger can be set to run on a day of the week or only once (today).
```

**Review by MartinHjelmare — CHANGES_REQUESTED** [review]

File: source/_integrations/wilight.markdown [hunk 7]
```diff
@@ -4,10 +4,12 @@ description: Instructions on how to integrate WiLight devices into Home Assistan
 ha_category:
   - Cover
   - Fan
+  - Irrigation
   - Light
+  - Switch
 ha_release: 0.115
 ha_config_flow: true
-ha_iot_class: Local Polling
+ha_iot_class: Local Push
```
Comment (MartinHjelmare) [inline source/_integrations/wilight.markdown]: The integration manifest says it's Local Polling.

Comment (leofig-rj) [reply source/_integrations/wilight.markdown]: Fixed!
Thanks.

**Review by MartinHjelmare — APPROVED** [review]
Thanks!

**Pull request merged**